tests/conftest.py

```diff
@@ -95,7 +95,13 @@ def pypi_port():
 @contextmanager
 def start_pypi_server(packages, port, pypi_fallback):
     port = str(port)
-    cmd = ('pypi-server', '-vvvvvv', '-i', '127.0.0.1', '-p', port)
+    cmd = (
+        'pypi-server', '-vvvvvv',
+        '-i', '127.0.0.1',
+        '-p', port,
+        # Default fallback is HTTP, which is no longer supported.
```

Comment:
maybe a todo to remove this once a new enough pypiserver is released? or install pypiserver from git temporarily?